README.md

```diff
@@ -2,3 +2,21 @@
 
 # pyret-ide
 UI Components that power the pyret ide
+
+## React Storybook
+
+To use React Storybook:
+
+```npm run storybook```
+
+Then open a browser window to localhost:9001
+
+To add to react storybooks you need to:
+
+        1. Write the stories in the src/components/stories/directory
+
+        2. Add that story's filename to the the config file located in the .storybook/directory
```

Comment:
use the markdown syntax for numbered lists and put backticks around directories and filenames